oauthproxy.go

```diff
@@ -946,9 +946,7 @@ func (p *OAuthProxy) addHeadersForProxying(rw http.ResponseWriter, req *http.Req
 		}
 
 		if len(session.Groups) > 0 {
-			for _, group := range session.Groups {
-				req.Header.Add("X-Forwarded-Groups", group)
-			}
+			req.Header.Add("X-Forwarded-Groups", strings.Join(session.Groups, ","))
```

Comment:
Do we want this as `req.Header["X-Forwarded-Groups"]` to match the convention set above in the other `X-Forwarded-*` request headers?

Either that or `Set` right? I don't think `Add` is correct here?